src/control/lib/control/pool.go

```diff
@@ -708,3 +711,23 @@ func PoolReintegrate(ctx context.Context, rpcClient UnaryInvoker, req *PoolReint
 
 	return nil
 }
+
+// ParsePolicy will parse the incoming policy name string and return
+// a policy index used in the DAOS engine.
+// Returns an error if string cannot be found in the map
+func ParsePolicy(stringPolicy string) (policy uint32, err error) {
+
+	var policyMap = map[string]drpc.PoolPolicy{
+		"default":           drpc.PoolPolicyDefault,
+		"io_size":           drpc.PoolPolicyIoSize,
+		"write_intensivity": drpc.PoolPolicyWriteIntensivity,
+	}
```

Comment:
you can collapse this into policy, found := map[string]drpc.PoolPolicy{...}[stringPolicy]